static/js/publisher/release/releasesState.test.js

```diff
@@ -0,0 +1,30 @@
+import { getPendingRelease } from "./releasesState";
```

Comment:
`releasesState` is a bunch of legacy pre-redux code that stays here because it was not yet migrated to more appropriate place.

Instead of adding test to it, we should probably move it somewhere where it belongs better.
Seems like functionality like that would work as helper or even a selector.